R/gdal_cmd_builder.R

```diff
@@ -327,14 +344,23 @@ gdal_cmd_builder <- function(executable,parameter_variables=c(),
 									collapse=" ")			
 						} else
 						{	
-							parameter_variables_repeatable_string <- paste(
-									paste(flag,
-											qm(parameter_values[[which(names(parameter_values)==X)]]),
-											sep=""),
-									collapse=" ")
+						  if (X %in% parameter_named) 
```

Comment:
Comment: `parameter_named` superseded by `parameter_noquotes`. I think this should be informed in documentation.

Reply:
Closed in c22df9d